### PR TITLE
Statoids import - admin0

### PR DESCRIPTION
**Pull request by stepps00**
This PR uses the "countries" (`admin0`) list from Statoids to update and add properties to the appropriate WOF records.

The script used to complete this work is here: https://github.com/mapzen/whosonfirst-toolbox/blob/master/scripts/statoids_import_admin0.py

Out of the 249 features (rows) in the Statoids file, 3 did not match to a record in WOF. These "leftover" features are either taken care of in the `admin1` import (like Christmas Island ([PR](https://github.com/whosonfirst-data/whosonfirst-data/pull/891))) or tracked in another issue ([Tokelau issue](https://github.com/whosonfirst-data/whosonfirst-data/issues/348)). 

Property edits only; no PIP work required.

### CODE REVIEW

**Comment by stepps00** [comment]
To do:

- [ ] Take care of three missing records...

File: data/136/251/281/136251281.geojson [hunk 1]
```diff
@@ -11,6 +11,7 @@
     "geom:longitude":-67.689731,
     "gp:id":0,
     "iso:country":"NL",
+    "itu:country_code":"599",
```
Comment (nvkelso) [inline data/136/251/281/136251281.geojson]: Please add link to PR for updated **whosonfirst-properties** documentation.

File: data/136/251/281/136251281.geojson [hunk 2]
```diff
@@ -109,7 +124,15 @@
     ],
     "wof:breaches":[],
     "wof:concordances":{
-        "gn:id":"7626844"
+        "fifa:id":"ANT",
```
Comment (nvkelso) [inline data/136/251/281/136251281.geojson]: Please add link to PR for updated **whosonfirst-sources** documentation for this and related new concordance values.

File: data/856/321/63/85632163.geojson [hunk 3]
```diff
@@ -12,6 +12,7 @@
     "gn:id":"",
     "iso:country":"GU",
     "iso:parent":"US",
+    "itu:country_code":"1-671",
```
Comment (nvkelso) [inline data/856/321/63/85632163.geojson]: This seems to also include an area code in addition to the country code?

File: data/856/321/69/85632169.geojson [hunk 4]
```diff
@@ -12,6 +12,7 @@
     "gn:id":"",
     "iso:country":"VI",
     "iso:parent":"US",
+    "itu:country_code":"1-340",
```
Comment (nvkelso) [inline data/856/321/69/85632169.geojson]: Ditto on country code + area code.

File: data/856/321/85/85632185.geojson [hunk 5]
```diff
@@ -375,16 +376,32 @@
         "naturalearth"
     ],
     "src:lbl:centroid":"mapshaper",
+    "statoids:dial":"1-721",
```
Comment (nvkelso) [inline data/856/321/85/85632185.geojson]: Ditto on country code + area code. But then it's part of the NL so maybe my understanding of how all this dial stuff works needs an upgrade.

Comment (nvkelso) [inline data/856/321/85/85632185.geojson]: Ah, Wikipedia to the rescue:

- https://en.wikipedia.org/wiki/List_of_country_calling_codes#Zone_1:_North_American_Numbering_Plan_Area



Propose `itu:region` as a single integer property in the space 1 to 9. Then the `itu:country_code` should really be a list and be whatever Statoids give us until we research it more.

So for this one:

```
"itu:region":1,
"itu:country_code": ["1-721"]
```

File: data/136/251/281/136251281.geojson (hunk 2 above)
Comment (stepps00) [inline data/136/251/281/136251281.geojson]: I've opened https://github.com/whosonfirst/whosonfirst-sources/issues/83 to track Statoids-related updates in the `whosonfirst-sources` repo.

File: data/136/251/281/136251281.geojson (hunk 1 above)
Comment (stepps00) [inline data/136/251/281/136251281.geojson]: I've opened https://github.com/whosonfirst/whosonfirst-properties/issues/32 to track Statoids-related updates in the `whosonfirst-properties` repo.

File: data/856/321/85/85632185.geojson (hunk 5 above)
Comment (stepps00) [inline data/856/321/85/85632185.geojson]: This all sounds good, I'll update the `itu` properties in a new commit.

Do you also expect the `statoids:dial` property to also be a list type?

Comment (nvkelso) [inline data/856/321/85/85632185.geojson]: No, that's just a big string of whatever they give us in the one cell.

**Comment by thisisaaronland** [comment]
Corcondances need to be singletons because applications should not have to guess at field types (lists versus strings). There is an open conversation about adding a `wof:concordances_alt` field that is a list.

**Comment by stepps00** [comment]
After review, the `wof:concordance_alt` property is not needed on any records in this PR.

There are "dual concordances" for several `admin0` records in the Stations table, but they correspond to different place types in WOF. An example is dual concordances for Palestine, but the concordance values actually represent our `region` records of the West Bank and Gaza Strip.

**Comment by nvkelso** [comment]
@stepps00 Can you document `wof:concordance_alt` in the properties repo, please? In a separate `whosonfirst-data` PR (for a rainy day) you could take say the Washington, DC locality feature that has placetype_alt list and add the concordance_alt from FIPS.

**Comment by stepps00** [comment]
Re: https://github.com/whosonfirst-data/whosonfirst-data/pull/893#issuecomment-322576334

I've opened https://github.com/whosonfirst/whosonfirst-properties/issues/34 to track that new property.

**Comment by nvkelso** [comment]
Other than that I'm good :)

**Comment by stepps00** [comment]
@nvkelso - the most recent commit ([107d52f](107d52f)) updated all records in this PR to include `"mz:is_current":1` (all records are current).

@thisisaaronland - this PR did not end up using the `"wof:concordances_alt"` property, but I opened whosonfirst/whosonfirst-properties#34 to track that property in the `whosonfirst-properties` repo.

I'll be running this PR through the pipeline shortly, unless there is anything else to add.

**Pull request merged**